solutions/swb-reference/integration-tests/support/utils/constants.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+//Average start waiting time is 5 minutes, setting start max waiting to 8 minutes
```

Comment:
Let's just double the max wait time for all of these. So in this, this can be 10 min.

Reply:
changed to 10 min